src/u2f.c

```diff
@@ -269,16 +291,51 @@ static void _register(const USB_APDU* apdu, Packet* out_packet)
         return;
     }
 
+    if (!memory_is_initialized()) {
+        if (!_nudge_label) {
```

Comment:
i'd move all of this (including _push) into _create_nudge_init_label, then all the component business is in neatly in one place.

Reply:
done